app/dashboard/router.py

```diff
@@ -252,6 +254,14 @@ def get_queryset(self):
             statuses = self.request.query_params.get('status__in').split(',')
             queryset = queryset.filter(idx_status__in=statuses)
 
+        if 'applicants' in param_keys:
```

Comment:
When the user selects "ALL" from the dropdown, the query parameter is set to "ALL", correct? How is this handled in the backend? If the idea is to just skip the filter, then we shouldn't add an annotation as it is bad for performance (especially with a `Count()`)

Reply:
@danlipert I've fixed it. Could you check?

Reply:
Looks good - thanks!